topics/transcriptomics/tutorials/goenrichment/tutorial.md

```diff
@@ -0,0 +1,343 @@
+- GO annotations, associating the genes in the population set to GO terms
+- The GO ontology, with the description of GO terms and their relationships
+ 
+For each GO term, we need to count the frequency (**k**) of genes in the study set (**n**) that are associated to the term, and the frequency (**K**) of genes in the population set (**N**) that are associated to the same term. Then we test how likely would it be to obtain at least **k** genes associated to the term if **n** genes would be randomly sampled from the population, given the frequency **K** and size **N** of the population.
+
+The appropriate statistical test is the one-tailed variant of Fisher’s exact test, also known as the hypergeometric test for over-representation. When the one-tailed version is applied, this test will compute the probability of observing at least the sample frequency, given the population frequency.  The [hypergeometric distribution](https://en.wikipedia.org/wiki/Hypergeometric_distribution) measures precisely the probability of **k** successes in **n** draws, without replacement, from a finite population of size **N** that contains exactly **K** successful objects: 
+> 
+![](../../images/goenrichment_formula.png)
+
+For this first exercise we will use data from [Trapnell et al. 2014](https://www.ncbi.nlm.nih.gov/pubmed/22383036 "Trapnell et al. data"). In this work, the authors created an artificial dataset of gene expression in *Drosophila melanogaster*, where 300 random genes were set (insilico) to be differentially expressed between two conditions.
+
+> ### {% icon hands_on %} Hands-on:
+> The data for this tutorial is available at [Zenodo](https://zenodo.org/record/1255038#.Wx4qTBwh3CI) to download.
+> 1. **Create a new history** 
+>
+> 2. **Upload to the Galaxy** the following files:
+>    - go.obo 
+>    - drosophila_gene_association.fb 
+>    - trapnellPopulation.tab 
+>
+>    > ### {% icon tip %} Tip: Upload data to Galaxy [<span style="color:red">[1]</span>](https://galaxyproject.github.io/training-material/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html) 
+>    > * **Click** on the upload button in the upper left of the interface.
+>    >
+>    > ![](../../images/goenrichment_galaxy_upload.png) 
+>    >
+>    > * Press **Choose local file** and search for your file.
+>    > * Press **Start** and wait for the upload to finish.
+>    {: .tip}
+>
+> 3. **Rename** the *go.obo* file to **GO** and *drosophila_gene_association.fb* file to **GO annotations Drosophila melanogaster**. 
+>
+>    > After you upload the files, and if you press the **eye icon** of *trapnellPopulation.tab* you should look someting like this:
+>    > ![](../../images/goenrichment_trapnellFile.png)
+> **Figure 2** Trapnell file
+>
+>    > As you can see, we only have the population file, containing all genes. We have to define our genes of interest.
+>
+> 4. In the tool menu, navigate to `Filter and Sort -> Filter data on any column using simple expressions`.
```

Comment:
I think you could omit this step (and move the comment) and just go straight to the Filter tool.